pkg/datapath/loader/base.go

```diff
@@ -255,6 +259,52 @@ func (l *Loader) Reinitialize(ctx context.Context, o datapath.BaseProgramOwner,
 		ret  error
 	)
 
```

Comment:
The next 2 functions can be deduped. Looks like the only different thing is the first argument of `unix.Socket()`.